syne_tune/constants.py

```diff
@@ -72,6 +72,20 @@
 """Name for ``upload_dir`` in ``RemoteTuner``"""  # pylint: disable=W0105
 
 
+# File names
+
+ST_RESULTS_DATAFRAME_FILENAME = "results.csv.zip"
```

Comment:
Would it make sense to add the fact that this contains intermediate results, to disambiguate from the final results JSON? 

Also, to check my understanding, this is written always, whereas the final results JSON is not always written? 

Question: at the end of tuning, would this `results.csv.zip` contain the same info as the final results JSON?

Reply:
No, the difference is not when these two are written (they are both written intermediately, so if things crash, you have something until then), but `results.csv` is a dataframe with a time index, so contains records of events during the experiment (e.g., when certain reports were obtained, ...). `final-results.json` is a set of results not indexed by time.

Reply:
Thanks for the clarification! Sounds good.